packages/destination-actions/src/destinations/launchdarkly-audiences/syncAudience/custom-audience-operations.ts

```diff
@@ -0,0 +1,116 @@
+import type { RequestClient } from '@segment/actions-core'
+import type { Payload } from './generated-types'
+
+import { CONSTANTS } from '../constants'
+import { Settings } from '../generated-types'
+
+type AudienceBatch = {
+  environmentId: string
+  batch: AudienceBatchItem[]
+}
+
+type AudienceName = string
+
+type AudienceBatchItem = {
+  userId: string
+  cohortName: AudienceName
+  cohortId: string
+  value: boolean
+}
+
+/**
+ *
+ * @param key snake-cased Audience key
+ * @returns sentence cased version of the Audience key (used for the LaunchDarkly Segment name)
```

Comment:
```suggestion
 * @returns sentence sentence-cased version of the Audience key (used for the LaunchDarkly Segment name)
```
or just `@returns sentence-cased version...`, I'm not sure if the first word is the var name of what's being returned or if this goes immediately into a description